testnet/stacks-node/src/neon_node.rs

```diff
@@ -87,9 +88,9 @@ struct MicroblockMinerState {
 enum RelayerDirective {
     HandleNetResult(NetworkResult),
     ProcessTenure(ConsensusHash, BurnchainHeaderHash, BlockHeaderHash),
-    RunTenure(RegisteredKey, BlockSnapshot),
+    RunTenure(RegisteredKey, BlockSnapshot, u128),
```

Comment:
Add comment for `RunTenure` and `RunMicroblockTenure` described what data is in tuple (like what is the `u128` for).